docs/product/releases/health/index.mdx

```diff
@@ -4,6 +4,12 @@ sidebar_order: 50
 description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
 ---
 
+<Note>
+
+Sessions as it pertains to Release Health are not billed and should not be confused with Session Replay. The video-like reproduction of the user's session.
+
+</Note>
```

Comment:
```suggestion
<Note>

Note, that Sessions (as described below) are not the same as a [Session Replay](/product/explore/session-replay/), (a video-like reproduction of the user's session). We don't bill for release health sessions.

</Note>
```
I'd move this note so that it lives just under the "Sessions" heading. It's a bit confusing at the top.